### DISCUSSION TITLE
Is there not support for "space-y-{amount}" ?

### DISCUSSION THREAD

**Discussion by joseph-ampfer** [jaredh159/tailwind-react-native-classnames#274] [Q&A]
I am following a tutorial that is using NativeWind. Everything was fine until they use space-y-4, and I get an error that it is unknown or invalid utility.

**Comment by jaredh159** [comment]
not sure why you're posting the question here then... NativeWind is a different project that does not use this library at all. could you maybe post this question over there? or am i misunderstanding why this is relevant?

**Reply by joseph-ampfer** [reply]
Sorry for the confusion, I am currently using twrnc in my project. I was following someone who was using NativeWind.

**Reply by jaredh159** [reply]
ah, ok. makes sense. so, NativeWind also supports RN for Web, which this library does not. There are many utilities that only work for web, but don't correspond to any style prop in RN non-web. I think space-y-XXX is one of those.  unless RN native recently added support for those.
